packages/blade/src/components/BottomNav/BottomNav.web.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import styled from 'styled-components';
+import BaseBox from '~components/Box/BaseBox';
+import { getStyledProps } from '~components/Box/styledProps';
+import { Text } from '~components/Typography';
+import { makeSpace } from '~utils';
+import { componentZIndices } from '~utils/componentZIndices';
+import { getFocusRingStyles } from '~utils/getFocusRingStyles';
+import { throwBladeError } from '~utils/logger';
+import { makeAccessible } from '~utils/makeAccessible';
+import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
+import type { BottomNavItemProps, BottomNavProps } from './types';
+
+/**
+ * ### BottomNav component
+ *
+ * Bottom navigation component is a persistent user interface element at the bottom of a mobile app screen, providing quick access to core functionalities through icons and labels.
+ *
+ * ---
+ *
+ * #### Usage
+ *
+ * ```jsx
+ * import { NavLink } from 'react-router-dom';
+ *
+ * <BottomNav>
+ *   <BottomNavItem
+ *     as={NavLink}
+ *     isActive={true}
+ *     title="Payments"
+ *     icon={WalletIcon}
+ *     href="/payments"
+ *   />
+ *   <BottomNavItem
+ *     as={NavLink}
+ *     title="Transactions"
+ *     icon={TransactionsIcon}
+ *     href="/transactions"
+ *   />
+ *   <BottomNavItem
+ *     title="More"
+ *     icon={MoreIcon}
+ *     onClick={() => {
+ *       // This can be used to open SideNav drawer
+ *       setIsSideNavOpen(true)
+ *     }}
+ *   />
+ * </BottomNav>
+ * ```
+ *
+ * Checkout {@link https://blade.razorpay.com/??path=/docs/components-bottomnav--doc BottomNav Documentation}
+
+ */
+const BottomNav = ({
+  children,
+  zIndex = componentZIndices.bottomNav,
+  testID,
+  ...styledProps
+}: BottomNavProps) => {
+  if (__DEV__) {
+    const childrenCount = React.Children.count(children);
+    if (childrenCount > 5 && childrenCount < 2) {
+      throwBladeError({
+        moduleName: 'BottomNav',
+        message: 'children cannot be less than 2 and more than 5',
+      });
+    }
+  }
+
+  return (
+    <BaseBox
+      role="navigation"
+      position="fixed"
+      bottom="spacing.0"
+      left="spacing.0"
+      elevation="midRaised"
+      width="100%"
+      backgroundColor="surface.background.gray.intense"
+      borderTopWidth="thin"
+      borderTopColor="surface.border.gray.muted"
+      paddingX="spacing.2"
+      display="flex"
+      flexDirection="row"
+      {...getStyledProps(styledProps)}
+      zIndex={zIndex}
+      {...metaAttribute({
+        testID,
+        name: MetaConstants.BottomNav,
+      })}
+    >
+      {children}
+    </BaseBox>
+  );
+};
+
+const StyledBottomNavItem = styled(BaseBox)<{ to?: string }>((props) => {
+  return {
+    textDecoration: 'none',
+    color: props.theme.colors.interactive.text.gray.subtle,
+    backgroundColor: 'transparent',
+    border: 'none',
+    paddingLeft: makeSpace(props.theme.spacing[0]),
+    paddingRight: makeSpace(props.theme.spacing[0]),
+    '&[aria-current="page"]': {
```

Comment:
No transition or animations when switching active tab? won't it look abrupt?

Reply:
Hmm yes I can check with RK once he is back. Will probably take it as enhancement if we end up going this route. I can add same transitions as SideNavLink meanwhile